### PR TITLE
Fix making boards in the admin panel

### PR DESCRIPTION
**Pull request by tiffz**
I want to add boards to prod. :D

### CODE REVIEW

File: saylua/modules/forums/models/db.py [hunk 1]
```diff
@@ -16,9 +38,12 @@ class Board(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     title = db.Column(db.String(256))
-    url_slug = db.Column(db.String(256))
+    text_id = db.Column(db.String(256), unique=True)
```
Comment (NoiSek) [inline saylua/modules/forums/models/db.py]: text_id is a weird name, m8y, there's an implication there that isn't necessarily true 
(Which is that text_id's are permanent, non-mutable identifiers)

**Review by NoiSek — APPROVED** [review]

File: saylua/modules/forums/models/db.py (hunk 1 above)
Comment (tiffz) [inline saylua/modules/forums/models/db.py]: Yeah, that's true. *tried to think of a better name*

**Review by LikeMyBread — APPROVED** [review]
Looks good to me, perhaps text_name might work? It seems redundant but I think it gets the idea across.

File: saylua/modules/forums/models/db.py (hunk 1 above)
Comment (tiffz) [inline saylua/modules/forums/models/db.py]: Hmm, what about "identifier"?

Comment (tiffz) [inline saylua/modules/forums/models/db.py]: Hmm, what about "identifier"?

**Pull request merged**